rest/data/util.go

```diff
@@ -48,3 +48,14 @@ func CopyProject(ctx context.Context, sc Connector, projectToCopy *model.Project
 	}
 	return apiProjectRef, nil
 }
+
+func CreateProject(ctx context.Context, sc Connector, projectRef *model.ProjectRef) (*model.ProjectRef, error) {
```

Comment:
I don't think we need this. Can't the route and the resovler already just use sc.CreateProject? Really the only bit that would be duplicated is getting the user. The reason this was necesssary for CopyProject is because it used multiple different kind of connectors.

Reply:
I think the thing that _can_ be done for this ticket is moving the getting and validating of the project input into CreateProject, but again we shouldn't need a new function here for it since we only need the ProjectConnector.

Reply:
yeah, I was questioning wether we need this as I was writing it. Since you're having the same though, I'll close it.